build/includes/terraform.mk

```diff
@@ -15,6 +15,9 @@
 # The GKE development cluster name
 GCP_TF_CLUSTER_NAME ?= agones-tf-cluster
 
+# the current project
+current_project := $(shell $(DOCKER_RUN) bash -c "gcloud config get-value project 2> /dev/null")
```

Comment:
This will _always_ use the project from the docker image, whereas before you could override the project if you wanted. I doubt folks were doing that, but maybe give a heads up in the developer slack channel when this goes in as an fyi.

Reply:
This still allows users to override GCP_PROJECT like they would have done previously. `current_project` is a placeholder for the default value.